setup.py

```diff
@@ -7,46 +7,45 @@
 import re
 from setuptools import setup
 
-with open('flask_meld/__init__.py', 'r') as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        f.read(), re.MULTILINE).group(1)
+with open("flask_meld/__init__.py", "r") as f:
+    version = re.search(
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE
+    ).group(1)
 
 HERE = pathlib.Path(__file__).parent
 README = (HERE / "README.md").read_text()
```

Comment:
awesome! my intuition was: 

```
with open('README.md', 'r', 'utf-8') as f:
    readme = f.read()
```
because I was wondering if it closes the file, but it does! https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text